gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystem.java

```diff
@@ -589,6 +589,24 @@ public FSDataInputStream open(Path hadoopPath, int bufferSize) throws IOExceptio
         });
   }
 
+  public FSDataInputStream open(FileStatus status) throws IOException {
```

Comment:
Is using `open` intentional? What about api metric counters given we already have an API with `open`  and I do see we use dame statistics i.e. `GhfsStatistic.INVOCATION_OPEN`

Reply:
yes. I do not think adding a new metric for this is required. Too many metrics are not good.

Reply:
But it's a different api all together. Not capturing metric for it is fine, but we should definitely not corrupt the existing metric.

Reply:
Discussed offline